src/AdminConsole/AuditLog/Loggers/AuditLoggerEfReadStorage.cs

```diff
@@ -0,0 +1,29 @@
+using AdminConsole.Db;
+using Microsoft.EntityFrameworkCore;
+using Passwordless.AdminConsole.AuditLog.DTOs;
+
+namespace Passwordless.AdminConsole.AuditLog.Loggers;
+
+public class AuditLoggerEfReadStorage : IAuditLoggerStorage
+{
+    private readonly ConsoleDbContext _context;
+
+    public AuditLoggerEfReadStorage(ConsoleDbContext context)
+    {
+        _context = context;
+    }
+
+    public async Task<IEnumerable<OrganizationEventDto>> GetOrganizationEvents(int organizationId, int pageNumber, int resultsPerPage) =>
```

Comment:
Default and max results, similar to API.

Reply:
A default is set on the `Organization/Log` page. Did we want it set down here at the db level?

Reply:
Nah probably not. As long as the user can't trigger a massive fetch.